pkg/controller/servicemesh/memberroll/controller.go

```diff
@@ -242,6 +242,10 @@ func (r *MemberRollReconciler) reconcileObject(ctx context.Context, roll *maistr
 		return reconcile.Result{}, err
 	} else if mesh == nil {
```

Comment:
not your change, but: indentation looks strange here?

Reply:
Why? Looks ok to me.

Reply:
nevermind it's my github side-by-side that makes it look broken